frontend/src/component/application/ApplicationEdit/ApplicationEdit.tsx

```diff
@@ -77,7 +76,7 @@ export const ApplicationEdit = () => {
     const tabData = [
         {
             label: 'Application overview',
-            component: <ApplicationView />,
+            component: <ApplicationOverview />,
```

Comment:
You probably only want this in the new component, right (Application.tsx). This is the old component and it's not hidden behind any flags. Of course, if it's for dev purposes, that's fine 😄